addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -9178,7 +9366,28 @@ registry.ColoredLevelBackground = registry.BackgroundToggler.extend({
      */
     onBuilt: function () {
         this._markColorLevel();
+        if (this.$target[0].tagName == "SECTION") {
+            this.trigger_up("option_update", {
+                optionName: "BackgroundShape",
+                name: "bgColorChanged",
+                data: { preview: false },
+            });
+        }
```

Comment:
Instead of relying on `"SECTION"`, send the current target via the data and compare in `notify`